.github/compatibility-test-matrices/release-v3.4.x/transfer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "chain-a": ["release-v3.4.x", "release-v6.0.x", "release-v5.1.x", "v5.0.1", "release-v4.2.x", "v4.1.1", "v3.3.1", "release-v2.5.x", "v2.4.2"],
+  "chain-b": ["release-v3.4.x", "release-v6.0.x", "release-v5.1.x", "v5.0.1", "release-v4.2.x", "v4.1.1", "v3.3.1", "release-v2.5.x", "v2.4.2"],
```

Comment:
in our v3.4.x I do we want to explicitly test again the tip of other release branches? I think it is fine to test only against all other released versions. The compatibility tests for those other branches should handle those cases.